integration_test/README.md

```diff
@@ -65,6 +68,11 @@ simplified to:
     no-op for some applications. If so, just leave the file empty.
 1.  `agent/ops-agent/linux/enable_<application>` to configure the Ops Agent to
     read the application's metrics exposed in the previous step.
+1.  (if necessary) `applications/<application>/exercise` to send some load to
```

Comment:
Should this step (line 71 - 72) be done in some earlier stage before the `expected_logs.yaml` and `metric_name.txt` checks?

Reply:
See my earlier comment; the view in github is deceptive. It does happen before expected_logs.yaml and metric_name.txt. But this section is just a list of what files to add, it doesn't have to be in order.

I rewrote some of this text as well.

Reply:
Ah I see. Yeah, I got confused by way the GitHub displays it. Now I get it. Thanks for the clarification!